source/develop/release-management/features/network/copy-host-networking.html.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Copy Host Networking
+category: feature
+authors: Ales Musil
+feature_name: Copy Host Networking
+feature_modules: copy-host-networking
+feature_status: In Development
+---
+
+# Copy Host Networking
+
+## Summary
+
+As it turned out setting up multiple hosts at once can be rather tedious via
+UI. The slow point is that the administrator has to go through every new host
+and do Setup Host Network manually. By utilizing Copy Host Networking, the administrator
+has to set only single host, for the rest the administrator can just specify the host
+to take the network configuration from. 
+
+
+### Owner
+
+- Feature Owner: Ales Musil (amusil)
+- E-mail: <amusil@redhat.com>
+
+
+### Benefit to oVirt
+
+The Copy Host Network helps the administrator with Day 1 deployment automation
+for host networking configuration across multiple hosts. This feature can also be
+on Day 2 for any additional host that needs the same configuration of networking. 
+
+### Example flow
+
+After creation of new Cluster, the administrator adds several hosts (`Host0, Host1, Host2`). 
+`Host0` networking has to be configured manually either by UI or through REST API.
+Other hosts, as long as they have same layout of network interfaces, can copy
+their settings from the host that has been already configured. 
+
+The copy can be done via UI. Going to hosts network interfaces and by clicking
+the `Copy Host Networking` button. In the popup window that will appear 
+the administrator can select from which host (`Host0`) should the configuration be copied.
+
+Via REST API every host has method called `copynetworks` which needs a single
+parameter, id of the host (`Host0`) that the configuration should be copied from. 
+
+### Limitation
+
+* The host configuration can be copied only within the same cluster. 
+
+* Configuration of static IP address for any of the networks cannot be copied. The static IP
+networks have their IP incremented by one to stay in range of the mask. 
```

Comment:
Just an idea:
Can we add an option may be to set a new static IP address during the copy network from host. nice to have.

Reply:
I will add this to the future work.

Reply:
Another idea: Can we have a parameter which switches between the proposed incrementing IPs to no copy static IP config?

Reply:
I don't see a use case for copying static IP from one host to another - it is an accident waiting to happen.
We have two options:
- copy only L2
- assign (what ovirt engine thing is) a free IP address in the subnet provided by the original host

the second option is really cool, but it has problems if some of the subnet is not managed by oVirt. I'd keep it to a later phase of this feature.